website/docs/guides/2.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,348 @@
+---
+layout: "azurerm"
+page_title: "Azure Active Directory: 2.0 Upgrade Guide"
+sidebar_current: "docs-azurerm-guide-2.0-upgrade"
+description: | -
+    This page documents how to migrate between two resources in the Azure Provider which have been renamed.
```

Comment:
Pretty sure this document talks about more than just this. Let's leave the description generic.

```suggestion
    Azure Resource Manager: 2.0 Upgrade Guide
```